docs/content/intro/protocol.md

```diff
@@ -10,7 +10,7 @@ The original vgo research paper on Download protocol can be found here: https://
 
 Each of these endpoints sits on top of a module. Let's assume module `htp` authored by `acidburn`.
 
-So for each of endpoints mentioned bellow we will assume address `acidburn/htp/@v/{endpoint}` (e.g `acidburn/htp/@v/list`)
+So for each of endpoints mentioned below we will assume address `acidburn/htp/@v/{endpoint}` (e.g `acidburn/htp/@v/list`)
```

Comment:
While your there, can you adjust this line to read `So for each of the endpoints..`?

Reply:
@robjloranger could we do that in a follow-up?

Reply:
(I'm gonna merge this and fix that up in a PR right after this